Drv/LinuxGpioDriver/LinuxGpioDriverCommon.cpp

```diff
@@ -0,0 +1,65 @@
+// ======================================================================
+// \title  LinuxGpioDriverImpl.cpp
+// \author tcanham
+// \brief  cpp file for LinuxGpioDriver component implementation class
+//
+// \copyright
+// Copyright 2009-2015, by the California Institute of Technology.
+// ALL RIGHTS RESERVED.  United States Government Sponsorship
+// acknowledged.
+//
+// ======================================================================
+
+#include <Drv/LinuxGpioDriver/LinuxGpioDriver.hpp>
+#include <FpConfig.hpp>
+
+namespace Drv {
+
+// ----------------------------------------------------------------------
+// Construction, initialization, and destruction
+// ----------------------------------------------------------------------
+
+LinuxGpioDriver ::LinuxGpioDriver(const char* const compName) : LinuxGpioDriverComponentBase(compName), m_fd(-1) {}
```

Comment:
You are already initializing `m_fd` in the header file. Remove `m_fd`'s initialization from here.